<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QtMaterialTabs" name="tabs" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>122</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboLanguages">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>98</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>361</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;You should disable hardware acceleration if you are running the app inside a virtual machine or on older hardware and you observe graphical glitches such as flickering or disappearing widgets. &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Requires app restart.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleAutoSave" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>22</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AutoSave:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>60</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open last file on startup:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleOpenLast" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleHardware" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>162</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>174</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hardware acceleration:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>22</y>
         <width>150</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Default unit:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>77</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Snap to Grid:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleSnapGrid" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>67</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleShowGrid" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>105</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>115</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Grid</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QtMaterialToggle" name="toggleShowRuler" native="true">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>220</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Ruler</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboDefaultUnit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QComboBox" name="comboStyle">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>13</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Style:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialToggle</class>
   <extends>QWidget</extends>
   <header>qtmaterialtoggle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialTabs</class>
   <extends>QWidget</extends>
   <header>qtmaterialtabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
